examples/grpc-service-basic-auth-file-user-store/grpc_service_basic_auth_file_user_store.md

```diff
@@ -1,27 +1,13 @@
 # gRPC service - Basic authentication file user store
 
-A gRPC service/resource can be secured with Basic authentication and optionally by enforcing authorization. Then, it validates the Basic Auth token sent as the `Authorization` metadata against the provided configurations. This reads data from a file, which has a TOML format. This stores the usernames, passwords for authentication, and scopes for authorization.
-
-Ballerina uses the concept of scopes for authorization. A resource declared in a service can be bound to one/more scope(s).
-
-In the authorization phase, the scopes of the service/resource are compared against the scope included in the user store for at least one match between the two sets.
-
-The `Config.toml` file is used to store the usernames, passwords, and scopes. Each user can have a password and optionally assigned scopes as an array.
+The `grpc:Service` can be secured with basic authentication and additionally, scopes can be added to enforce authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` metadata against the provided configurations in the `Config.toml` file. This file stores the usernames and passwords for the authentication and the scopes for the authorization. To engage authentication, set the default values for the `fileUserStoreConfig` field and add the `Config.toml` file next to the service file. To engage authorization, set scopes to the `scopes` field. Both configurations must be given as part of the service configuration. A `grpc:UnauthenticatedError` is sent to the client when the authentication fails, and a `grpc:PermissionDeniedError` is sent to the client when the authorization fails. Use this to authenticate and authorize requests based on user stores.
```

Comment:
```suggestion
The `grpc:Service` can be secured with basic authentication and additionally, scopes can be added to enforce authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` metadata against the configurations provided in the `Config.toml` file. This file stores the usernames and passwords for the authentication and the scopes for the authorization. To enable authentication, set the default values for the `fileUserStoreConfig` field and add the `Config.toml` file next to the service file. To enable authorization, set the scopes to the `scopes` field. Both configurations must be given as part of the service configuration. A `grpc:UnauthenticatedError` is sent to the client when the authentication fails, and a `grpc:PermissionDeniedError` is sent to the client when the authorization fails. Use this to authenticate and authorize requests based on user stores.
```
Too lengthy. Consider breaking the paragraph into two.